MSAL/test/unit/native_auth/network/responses/validator/MSALNativeAuthSignUpResponseValidatorTests.swift

```diff
@@ -68,25 +68,25 @@ final class MSALNativeAuthSignUpResponseValidatorTests: XCTestCase {
     func test_whenSignUpStart_verificationRequiredErrorWithSignUpTokenAndUnverifiedAttributes_it_returns_verificationRequired() {
         let error = createSignUpStartError(
             error: .verificationRequired,
-            signUpToken: "sign-up token",
+            continuationToken: "sign-up token",
```

Comment:
There are a couple of `sign-up token` in this file.

Reply:
Thanks, all strings updated